packages/client/src/views/CorrectionForm/VerifyCorrector.tsx

```diff
@@ -109,12 +109,16 @@ class VerifyCorrectorComponent extends React.Component<IFullProps> {
       const iDType = (info[fields.identifierTypeField] ||
         info[fields.identifierOtherTypeField]) as string
 
-      const firstNames = info[
-        fields.nameFields[intl.locale].firstNamesField
-      ] as string
-      const familyName = info[
-        fields.nameFields[intl.locale].familyNameField
-      ] as string
+      const firstNameIndex = fields.nameFields[intl.locale]
+        ? fields.nameFields[intl.locale].firstNamesField
+        : fields.nameFields[intl.defaultLocale].firstNamesField
+
+      const familyNameIndex = fields.nameFields[intl.locale]
+        ? fields.nameFields[intl.locale].familyNameField
+        : fields.nameFields[intl.defaultLocale].familyNameField
+
+      const firstNames = info[firstNameIndex] as string
+      const familyName = info[familyNameIndex] as string
```

Comment:
I think you already solves your problem here. So why have you needed to add the following font?
```
fr: {
      notosans: {
        normal: 'NotoSans-Light.ttf',
        bold: 'NotoSans-Regular.ttf'
      }
```

Reply:
font part is needed for the 3rd issue of the ticket, because font for french was undefined when this part is called 
```
return pdfMake.createPdf(
    JSON.parse(definitionString),
    undefined,
    template.fonts[intl.locale]
  )
```